### DISCUSSION TITLE
Plano de saúde

### DISCUSSION THREAD

**Discussion by alvesmog** [DyegoCosta/trabalhando-remoto#191] [General]
Pessoal, estou criando esta thread para centralizar informações a cerca de plano de saúde para autônomos. Vi que havia uma outra thread sobre o assunto, mas o autor fechou a issue.

Migrei de CLT para trabalhar como contractor para o exterior e estou em busca de opções, a plataforma que recebo (deel) oferece 2 planos: alice e safety wing. Alguém já ouviu falar?

Pelo que entendi esse Alice é uma startup brasileira e o safety wing é um seguro para nomades digitais, aparentemente a cobertura que eles oferecem é completa, nao sei até que ponto dá pra utilizar esse seguro de forma contínua

**Comment by lima-emanuel** [comment]
Eu sou contractor e faz 1 mês que tenho Sami. Usei muito pouco. Minha experiência foi regular, nem muito boa nem muito ruim. A decisão de usar Sami foi porque não tem cooparticipação e aceita PJs com 1 vida apenas.

**Comment by copercini** [comment]
O banco Inter tem algumas opções de planos de saúde dentro do app, não achei nada interessante na minha região, mas talvez seja útil para alguém

**Comment by alvesmog** [comment]
Outra opção, pra quem tiver bacharelado, é se filiar ao conselho da classe. 

No meu caso (engenharia), pelo que pesquisei dá pra fazer sul-america, bradesco, unimed e amil... dependendo da região. Cada conselho regional tem suas próprias regras.

**Comment by igorlvicente** [comment]
Eu tenho Unimed mas é estadual, então acho que nem adianta muito. Pelo que entendi você quer um que possa ser usado em qualquer lugar do mundo, é isso?

**Comment by yuriploc** [comment]
Generalizando um pouco mais a pergunta, plano de saúde como autônomo compensa mais do que como CPF?

**Comment by alvesmog** [comment]
> Eu tenho Unimed mas é estadual, então acho que nem adianta muito. Pelo que entendi você quer um que possa ser usado em qualquer lugar do mundo, é isso?

Um nacional já me atende muito bem. Não faço tantas viagens internacionais pra compensar contratar um com worldwide coverage 100% do tempo, geralmente contrato um seguro viagem pra cobrir durante o tempo que vou passar fora mesmo...


> Generalizando um pouco mais a pergunta, plano de saúde como autônomo compensa mais do que como CPF?

Pelas cotações que fiz, por CPF dificilmente compensa. Mas só falei com 3 corretores

**Comment by igorlvicente** [comment]
> Generalizando um pouco mais a pergunta, plano de saúde como autônomo compensa mais do que como CPF?

No meu caso, sim, compensa mais.
Não vi outros planos porque Unimed é o principal aqui onde moro. Eles têm até hospital próprio.
Se eles não aceitarem apenas 1 no CNPJ, coloque a esposa/parentes como dependentes.

**Comment by hsmiranda** [comment]
Tem bastante opções lá no banco inter, se você tiver conta neles da uma olhada que as vezes tu acha algo interessante.

**Comment by samiraguiar** [comment]
Vi que a Contabilizei tem algumas parcerias com operadoras, alguém já fez plano por lá?

**Comment by gabrielponto** [comment]
Já cotei com eles, acho que não tem muita diferença de cotar com outras
corretoras.

Em qui., 13 de jan. de 2022 às 10:47, Samir Aguiar ***@***.***>
escreveu:

**Comment by yagodorea** [comment]
@alvesmog estou num processo de onboarding aqui pela Deel e eles estão oferecendo agora plano de saúde Amil, saca só:

[Deal health Brazil.pdf](https://github.com/DyegoCosta/trabalhando-remoto/files/7905784/Deal.health.Brazil.pdf)
